Ansible/roles/marvin/files/smoke/test_vm_life_cycle.py

```diff
@@ -36,6 +36,7 @@
 from marvin.lib.common import (get_domain,
                                 get_zone,
                                 get_template,
+                                get_test_template,
                                list_hosts)
```

Comment:
most indentations look good now, not sure about this one. it is an import and I am not aware of any special rules for those but it looks weird to me that the middle three have a different level compared to the first and last. If it passes pep8 it's fine though, just an observation.